docs/docs/network-api-migration.md

```diff
@@ -0,0 +1,697 @@
+      "mode": "container",
+      "name": "first"
+    },
+    {
+      "mode": "container",
+      "name": "second"
+    }
+  ]
+}
+</pre>
+</td>
+</tr>
+</tbody>
+</table>
+
+#### App with bridge network
+
+<table class="table">
+  <tbody>
+  <tr>
+  <th>
+    <b>1.4 API</b>
+  </th>
+  <th>
+    <b>1.5 API</b>
+  </th>
+  </tr>
+  <tr>
+    <td>
+      <pre>
+{
+  "id": "bridge-net-old",
+  "cpus": 1,
+  "mem": 128,
+  "instances": 1,
+  "container": {
+    "type": "DOCKER",
+    "docker": {
+      "image": "nginx",
+      "network": "BRIDGE",
+      "portMappings": [
+        {
+          "containerPort": 80,
+          “hostPort”: 0,
+          "protocol": "tcp",
+          "name": "web"        }
+      ]
+    }
+  }
+}
+</pre>
+</td>
+
+<td>
+ <pre>
+ {
+  "id": "bridge-net-new",
+  "cpus": 1,
+  "mem": 128,
+  "instances": 1,
+  "container": {
+    "type": "DOCKER",
+    "docker": {
+      "image": "nginx"
+    },
+    "portMappings": [
+      {
+        "containerPort": 80,
+        "hostPort": 0,
+        "protocol": "tcp",
+        "name": "web"
+      }
+    ]
+  },
+  "networks": [
+    {
+      "mode": "container/bridge"
+    }
+  ]
+}
+</pre>
+</td>
+</tr>
+</tbody>
+</table>
+
+#### App with host network
+
+<table class="table">
+  <tbody>
+  <tr>
+  <th>
+    <b>1.4 API</b>
+  </th>
+  <th>
+    <b>1.5 API</b>
+  </th>
+  </tr>
+  <tr>
+    <td>
+      <pre>
+{
+  "id": "host-net-old",
+  "cpus": 1,
+  "mem": 128,
+  "instances": 1,
+  "container": {
+    "type": "DOCKER",
+    "docker": {
+      "image": "nginx",
+      "network": "HOST"
+    }
+  },
+  "portDefinitions": [
+    {
+      "port": 0,
+      "protocol": "tcp",
+      "name": "web"
+    }
+  ]
+}
+</pre>
+</td>
+
+<td>
+ <pre>
+ {
+  "id": "host-net-new",
+  "cpus": 1,
+  "mem": 128,
+  "instances": 1,
+  "container": {
+    "type": "DOCKER",
+    "docker": {
+      "image": "nginx"
+    }
+  },
+  "portDefinitions": [
+    {
+      "port": 0,
+      "protocol": "tcp",
+      "name": "web"
+    }
+  ],
+  "networks": [
+    {
+      "mode": "host"
+    }
+  ]
+}
+</pre>
+</td>
+</tr>
+</tbody>
+</table>
+
+### Invalid Definitions
+
+#### Mixed Networks (Invalid)
+You cannot combine network types.
+
+<table class="table">
+  <tbody>
+  <tr>
+  <th>
+    <b>1.5 API</b>
+  </th>
+  </tr>
+  <tr>
+    <td>
+      <pre>
+{
+  "id": "x-mixed-networks",
+  "cpus": 1,
+  "mem": 128,
+  "instances": 1,
+  "container": {
+    "type": "DOCKER",
+    "docker": {
+      "image": "nginx"
+    }
+  },
+  "portDefinitions": [
+    {
+      "port": 0,
+      "protocol": "tcp",
+      "name": "web"
+    }
+  ],
+  "networks": [ // Invalid because network modes must be identical
+    { "mode": "host" },
+    { "mode": "container/bridge" }
+  ]
+}
+</pre>
+</td>
+</tr>
+</tbody>
+</table>
+
+
+#### Missing container network name (Invalid)
+
+You must supply a `name` property to use a container network unless you have configured Marathon with the `default_network_name` flag set.
```

Comment:
I think you're using "set" as a verb here, but "flag set" may be construed as noun. Consider rephrasing.

Also, maybe provide a link to the command line flags doc